charts/argocd-applicationset/values.yaml

```diff
@@ -18,6 +18,9 @@ args:
   metricsAddr: :8080
   # -- The default health check port
   probeBindAddr: :8081
+  # -- The default webhook port
+  # hardcoded for now in the applicationset-controller
+  webhookAddr: :7000
```

Comment:
@mkilchhofer I saw you commented the PR implementing the hardcoded value of this. should we leave as is here ?

Reply:
I think we can remove it for now as there is no option to configure it.

Reply:
ok and done.